torchci/lib/GeneralUtils.ts

```diff
@@ -16,8 +16,13 @@ class ErrorWithStatusCode extends Error {
 
 export function includesCaseInsensitive(
   value: string,
-  pattern: string
+  pattern: string,
+  useRegex: boolean
 ): boolean {
+  if (useRegex) {
+    const regex = new RegExp(pattern, "i");
```

Comment:
this can throw. is the exception handled somewhere?

Reply:
Added a try catch, doesn't show the error except in console tho, but at least the page won't crash